src/js/modules/utilities/IOBuf.ts

```diff
@@ -190,7 +210,7 @@ export class IOBuf {
       fragment.used += auxReserve.length;
       reserves.push(auxReserve);
       if (size > 0) {
-        fragment = this.appendNewFragment(fragment.getSize());
```

Comment:
I think the previous name communicates better what the new function does. Most list implementations do this too: if there's available capacity, they add the given element. Otherwise, they "expand" the list and add the new element.

Reply:
I thinking is more explicit this name because `appendNewFragment` is like push a new fragment, while `getNextFragmentOrAppend` said that we try to get first the next fragment, otherwise we append a new one, also I don't change the function name, I create a new one if I use this name here, I need to change the current `appendNewFragment`